config/ingest.go

```diff
@@ -2,18 +2,20 @@ package config
 
 const (
 	defaultLinkCacheSize   = 1024
-	defaultLinkedChunkSize = 100
+	defaultLinkedChunkSize = 16384
```

Comment:
For the readers of code, can I ask you to add a comment explaining why this number?

Reply:
Done